iothub/device/src/InternalClient.cs

```diff
@@ -1225,26 +1225,26 @@ public async Task SetDesiredPropertyUpdateCallbackAsync(DesiredPropertyUpdateCal
         /// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
```

Comment:
Let's update this in a few ways:

1. Summary sets you can use it to set or clear/remove the callback.
2. For the second sentence, move it to `<remarks>`.